cmd/gateway-sia-anonymous.go

```diff
@@ -0,0 +1,44 @@
+package cmd
```

Comment:
Missing license header..

Reply:
Yes @dvstate it needs to say `Minio Cloud Storage` and not `David Gore`.  Will check back with @abperiasamy on this for final call.

Reply:
Fixed in most recent commit.